plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -1546,17 +1551,48 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
 
     toBuiltinMeaning _semvar NullList =
         let nullListDenotation :: SomeConstant uni [a] -> BuiltinResult Bool
-            nullListDenotation (SomeConstant (Some (ValueOf uniListA xs))) = do
+            nullListDenotation (SomeConstant (Some (ValueOf uniListA xs))) =
                 case uniListA of
-                    DefaultUniList _ -> pure $ null xs
-                    _                ->
-                        -- See Note [Structural vs operational errors within builtins].
-                        throwing _StructuralUnliftingError "Expected a list but got something else"
+                    DefaultUniList _argUni -> pure $ null xs
```

Comment:
... except that my personal best practice is to prefer `_` ("something is discarded") over the `_foo` ("foo is discarded"). This is because I've been bitten by discarding info in the past and decided to be more cautious about it.

Reply:
OK, it's fine, I will make a willful decision not to nitpick on that one.